packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -234,7 +235,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         :
         <Callout
           isBeakVisible={ false }
-          className={ css('ms-Dropdown-callout', styles.callout) }
+          className={ css('ms-Dropdown-callout', styles.callout,
+            calloutProperties ? calloutProperties.className : undefined) }
```

Comment:
@micahgodbolt: You were thinking we'd somehow mix in all of the properties provided, right? Not just `className`. Do we have an example somewhere else of where we're doing that?